client-src/elements/chromedash-feature-page_test.js

```diff
@@ -237,6 +237,16 @@ describe('chromedash-feature-page', () => {
     // spec link is clickable
     assert.include(specSection.innerHTML, 'href="fake spec link"');
 
+    const consensusSection = component.shadowRoot.querySelector('section#consensus');
+    assert.exists(consensusSection);
+    // spec link is clickable
+    assert.include(consensusSection.innerHTML, '<chromedash-vendor-views ');
+    assert.include(consensusSection.innerHTML, 'fake ff view text');
+    assert.include(consensusSection.innerHTML, 'href="fake ff url"');
+    assert.include(consensusSection.innerHTML, 'fake safari view text');
+    assert.include(consensusSection.innerHTML, 'href="fake safari url"');
+    assert.include(consensusSection.innerHTML, 'fake webdev view text');
```

Comment:
The web dev section always shows. (Or at least there is no conditional present like the firefox and Safari cases)

Add a separate comment saying WebDev text is present. This will be helpful since it is not a link